docs/training_guides/etl_best_practices.rst

```diff
@@ -0,0 +1,558 @@
+     ...
+
+There are more than five rows in our table, but ``print`` only displays the first five rows by default, for readability's sake.
+
+As you can see, this data corresponds to what's in the Google sheet. We display the data in a Python dictionary, with the column names as keys and the actual contents of each cell as the values. You can ask for any row of a Parsons Table as a dictionary::
+
+    print(attendance_records[0])
+    >> {'id': '46273', 'event_id': '454545', 'event_title': 'January Canvass', 'timeslot_id': '738375', 'timeslot_start_date': '1642865400', 'timeslot_end_date': '1642872600', 'status': 'REGISTERED', 'attended': 'true', 'person': '{"id": 1, "given_name": "Lou", "family_name": "Slainey", "email_address": "lslainey0@unicef.org", "phone_number": "3271326753", "postal_code": "78737"}'}
+
+You can also get any column of a Parsons Table as a list of values::
+
+    print(attendance_records["event_title"])
+    >> ['January Canvass', 'January Textbank', 'February Canvass', 'February Phonebank', 'March Relational Organizing Hour' ...
+
+Because individual rows are treated as dictionaries, and individual columns as list, that makes it easy to iterate over them with a for loop::
+
+    for index, attendance in enumerate(attendance_records):
+        print(attendance['person'])
+
+There are also a couple of convenience methods for getting the total number of rows and the list of column names::
+
+    attendance_records.num_rows
+    attendance_records.columns
+
+No matter where you got your data from, these methods should always work! That's the benefit of using a standardized format like a Parsons Table.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Transforming Data with Parsons
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+####################
+Fixing Dates + Times
+####################
+
+Let's make some fixes to our data. First off, those timeslot fields are confusing! What kind of date is ``1642865400``?
+
+(It's actually something called a `unix timestamp <https://www.unixtimestamp.com/>`_, which measures the total number of seconds since January 1st, 1970. Why January 1st, 1970? No real reason! They just needed to pick a date and I guess that seemed like a good one.)
+
+Let's convert these unix timestamps to something more readable. To do this, we define a function that takes in a value and returns a value::
+
+    def convert_to_legible_date(unix_date):
+        return datetime.utcfromtimestamp(int(unix_date)).strftime('%Y-%m-%d %H:%M:%S')
+
+Here, we're using the ``datetime`` library mentioned above. The ``strftime`` method is what determines the new format. For example, ``%Y`` means "Year with century as a decimal number" (like, say, 1970), and ``%m`` means "Month as a zero-padded decimal number" (like, say, 01). Here's a `cheatsheet <https://strftime.org/>`_ in case you want to play around with the formatting.
+
+Once we've got our function, we can apply it to all the rows in a column by using the `convert_column` function::
```

Comment:
Maybe here you could say, "by using the parsons table 'convert_column' function" just so it's clear it's a parsons method operating on the parsons table.